scripts/gemoji/gemoji.py

```diff
@@ -0,0 +1,43 @@
+#
```

Comment:
nit: can you add the shebang here?

Reply:
also - how did we as an engineer culture arrive at that word to describe this pattern